workloads/ingress-perf/run.sh

```diff
@@ -18,6 +18,24 @@ TOLERANCY=${TOLERANCY:-20}
 OS=$(uname -s)
 HARDWARE=$(uname -m)
 
+export TUNING_PATCH=${TUNING_PATCH:-"'{\"spec\":{\"nodePlacement\": {\"nodeSelector\": {\"matchLabels\": {\"node-role.kubernetes.io/infra\": \"\"}}}, \"replicas\": 2}}'"}
```

Comment:
Rather than modifying `run.sh`, can you push this changes into a new file  `custom-run.sh`? that way we won't touch any of the current logic in CPT

Reply:
I have used new configurations to run different replica numbers. I find t is more easier than custom-run.sh for me that I can just provide the new config file as an env var, no need to update the CI to use a new cmd like custom-run.sh. So I think I can close this one. 

@rsevilla87  If you have any suggestion to
https://github.com/cloud-bulldozer/e2e-benchmarking/pull/634#issuecomment-1757794801, it could be helpful. Now to tune the thread number, I do it outside of ingress-perf with oc command before the ingress-perf test.